docs/docusaurus/docs/core/set_up_a_gx_environment/install_gx.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Install GX
+---
+import TabItem from '@theme/TabItem';
+import Tabs from '@theme/Tabs';
+
+import PrereqPythonInstalled from '../_core_components/prerequisites/_python_installation.md';
+import InstallLocal from './_install_gx/_local_installation.md';
+import InstallEmrSpark from './_install_gx/_emr_spark_installation.md';
+import InstallDatabricks from './_install_gx/_databricks_installation.md';
+import InstallGxCloud from './_install_gx/_gx_cloud_installation.md';
+
+import PythonVersion from '../_core_components/_python_version.md';
+
+Great Expectations (GX) is a Python library that provides a framework for describing data with expressive tests and then validating that the data meets those criteria.
```

Comment:
```suggestion
Great Expectations (GX) is a Python library that provides a framework for describing data with expressive tests and then validating that the data meets those criteria.
```
I would recommend a rewrite of this text. It doesn't scan easily.

Reply:
Rewritten to correspond more to the purpose of this guide and less to defining what GX _is_ -- that part can remain in the "About GX" and "GX overview" topics.